test/adapter/json_api/relationship_test.rb

```diff
@@ -151,11 +151,8 @@ def test_relationship_with_everything
         private
 
         def test_relationship(expected, params = {})
-          options = params.fetch(:options, {})
-          links = params.fetch(:links, {})
-          meta = params[:meta]
           parent_serializer = AuthorSerializer.new(@author)
-          relationship = Relationship.new(parent_serializer, @serializer, options, links, meta)
+          relationship = Relationship.new(parent_serializer, @serializer, nil, params)
```

Comment:
passing a nil option is kind of a smell... I don't really get the difference between `serializable_resource_options` (`render_options`?)  and args/params